phpstan-baseline.neon

```diff
@@ -1303,6 +1303,11 @@ parameters:
 			count: 3
 			path: src/Type/ObjectType.php
 
+		-
+			message: "#^Method PHPStan\\\\Type\\\\ObjectType\\:\\:getReferencedClasses\\(\\) should return array\\<class\\-string\\> but returns array\\<int, string\\>\\.$#"
```

Comment:
baselined this error, as I would otherwise have to add a `@param class-string` to `ObjectType->construct()`, which would be a BC break